### PR TITLE
update todo

### PR DESCRIPTION
**Pull request by zhouzilong2020**
Line Chart Example:
截屏2022-05-16 10 08 35

Stacked Col Chart Example:
截屏2022-05-16 10 07 37

Bubble Chart Example:
截屏2022-05-16 10 08 03

### CODE REVIEW

File: data/sample.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [
   {
     "id": 0,
+    "externalId": "AAA",
```
Comment (tianzhou) [inline data/sample.json:4]: just curious, is this the actual external id or the id we come up?

File: README.md [hunk 2]
```diff
@@ -27,7 +27,11 @@ The market lacks a tool for developers to compare different database products be
   * [x] Data Refinement Menu
   * [x] Table for checked Instance
   * [ ] RAM / CPU wise calculator special for GCP
-* [ ] SEO with Nuxt (**On Going**) 
+* [ ] Cost Charts (**On Going**)
```
Comment (tianzhou) [inline README.md:30]:
```suggestion
* [ ] Cost Charts (**Ongoing**) 
```

File: README.md [hunk 3]
```diff
@@ -27,7 +27,11 @@ The market lacks a tool for developers to compare different database products be
   * [x] Data Refinement Menu
   * [x] Table for checked Instance
   * [ ] RAM / CPU wise calculator special for GCP
-* [ ] SEO with Nuxt (**On Going**) 
+* [ ] Cost Charts (**On Going**) 
+  * [ ] Compare the difference in monthly price between different offers ( Line Chart )
+  * [ ] Compare the difference in total price between different offers ( Accumulate Bar Chart )
+  * [ ] Compare the spec of different instance ( Scatter Chart )
+* [ ] SEO (**On Going**)
```
Comment (tianzhou) [inline README.md:34]:
```suggestion
* [ ] SEO (**Ongoing**) 
```

File: README.md [hunk 4]
```diff
@@ -27,7 +27,11 @@ The market lacks a tool for developers to compare different database products be
   * [x] Data Refinement Menu
   * [x] Table for checked Instance
   * [ ] RAM / CPU wise calculator special for GCP
-* [ ] SEO with Nuxt (**On Going**) 
+* [ ] Cost Charts (**On Going**) 
+  * [ ] Compare the difference in monthly price between different offers ( Line Chart )
+  * [ ] Compare the difference in total price between different offers ( Accumulate Bar Chart )
```
Comment (tianzhou) [inline README.md:32]: Not sure, what does accumulate bar chart mean. I thought it was just a simple bar char with instance spec as the x label?

Comment (zhouzilong2020) [reply README.md:32]: The bar has several dimension, like up-front commitment and on demand fee with different available rate.

File: data/sample.json (hunk 1 above)
Comment (zhouzilong2020) [reply data/sample.json]: Just some random ID I pick up.

**Comment by zhouzilong2020** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**